lib/web/desktop.go

```diff
@@ -104,7 +104,10 @@ func (h *Handler) createDesktopConnection(
 
 	sendTDPError := func(ws *websocket.Conn, err error) error {
```

Comment:
Do we need to maintain both `sendTDPError` and `sendTDPNotification`, or can a fatal notification take the place of `sendTDPError`?